src/LibraryViewExtension/web/layoutSpecs.json

```diff
@@ -1150,6 +1150,30 @@
               "path": "DSCore.String.Join"
             }
           ],
+          "childElements": [ ]
+        }
+      ]
+    },
+    {
+      "text": "Modifiers",
+      "iconUrl": "",
+      "elementType": "category",
+      "include": [],
+      "childElements": [
+        {
+          "text": "Geometry Color",
+          "iconUrl": "",
+          "elementType": "group",
+          "include": [
+            {
+              "path": "Modifiers.GeometryColor.ByGeometryColor",
```

Comment:
I think Modifiers as a namespace doesn't make sense unless we know what kind of objects are modified. So we can think about `Dynamo.Geometry.Modifiers.GeometryColor` or `Dynamo.Display.Modifiers.GeometryColor`.